source/grid/grid_out.cc

```diff
@@ -1451,8 +1458,8 @@ GridOut::write_xfig(const Triangulation<2> &tria,
             cell->vertex(GeometryInfo<dim>::ucd_to_deal[k % nv]);
           for (unsigned int d = 0; d < static_cast<unsigned int>(dim); ++d)
             {
-              int val = (int)(1200 * xfig_flags.scaling(d) *
-                              (p(d) - xfig_flags.offset(d)));
+              int val =
+                1200 * xfig_flags.scaling(d) * (p(d) - xfig_flags.offset(d));
```

Comment:
can you re-add the cast here? the right hand side is a floating point number and I think it's useful for a reader to see it being cast explicitly